lta/config.py

```diff
@@ -2,15 +2,19 @@
 
 import os
 
+NA = (0,) # use a tuple so 'is' operator works correctly
```

Comment:
I didn't know if this should be renamed for better external usage:

```Python 3
my_config = {
    "REST_HOST": "localhost",
    "REST_PORT": config.ENV_REQUIRED
}
my_config = config.from_environment(my_config)
```

But it seems like overkill. `NA` works well enough.

Reply:
Resolving this by making NA private.